src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -86,6 +122,9 @@ export default {
 			}
 			return this.promoData?.promoFund?.displayName ?? null;
 		},
+		isMatchingCampaign() {
+			return this.pageSettingData?.matcherAccountId !== undefined;
```

Comment:
This information will not be available here. Please just pass down a new prop into this component for `isMatchingCampaign` from the CCLanding so we don't duplicate the logic here.

Reply:
What I've suggested will make the other check in `campaignPartnerName` also function correctly which is currently referencing a missing prop...